crates/astria-sequencer/src/app.rs

```diff
@@ -715,6 +840,11 @@ impl App {
 
         // Get the latest version of the state, now that we've committed it.
         self.state = Arc::new(StateDelta::new(storage.latest_snapshot()));
+        self.app_hash = app_hash
+            .0
+            .to_vec()
+            .try_into()
+            .expect("root hash to app hash must succeed; both are 32 bytes");
 
         app_hash
```

Comment:
Seems like no callsite really needs the app_hash - could we change this method to not return it?